README.md

```diff
@@ -22,11 +22,11 @@ A full example leveraging other community modules is contained in the [examples/
 
 ```hcl
 data "aws_eks_cluster" "cluster" {
-  name = module.eks.cluster_id
+  name = module.my-cluster.cluster_id
 }
 
 data "aws_eks_cluster_auth" "cluster" {
-  name = module.eks.cluster_id
+  name = module.my-cluster.cluster_id
```

Comment:
But it's called `my-cluster` on line 40?

```
module "my-cluster" {
```

Reply:
Yep. That's why @AmitThakkar fixed the referenced module name in datasources. Or you have to name your module `module "eks" { ... }` on line 40 to make the exemple works.

Reply:
Ah yes, I must have been half asleep.